vendor/knative.dev/eventing/test/rekt/features/broker/control_plane.go

```diff
@@ -47,22 +48,22 @@ import (
 	triggerresources "knative.dev/eventing/test/rekt/resources/trigger"
 )
 
-func ControlPlaneConformance(brokerName string) *feature.FeatureSet {
+func ControlPlaneConformance(brokerName string, brokerOpts ...manifest.CfgFn) *feature.FeatureSet {
```

Comment:
There are some limitations in the upstream broker conformance tests. As a quick hack, I edited the files in vendor directory. Once PR passes fine, I will create a PR in knative/eventing for addressing those limitations.

Reply:
Created https://github.com/knative/eventing/pull/6510